Gordon360/Services/ProfileService.cs

```diff
@@ -79,24 +79,49 @@ public IEnumerable<AdvisorViewModel> GetAdvisors(string id)
             // Stored procedure returns row containing advisor1 ID, advisor2 ID, advisor3 ID 
             var idResult = RawSqlQuery<ADVISOR_SEPARATE_Result>.query("ADVISOR_SEPARATE @ID", idParam).FirstOrDefault();
 
+            //Obeject Array to store advisors information
+            AdvisorViewModel[] advisorsID = new AdvisorViewModel[3];
```

Comment:
Don't create a max-size array and leave some null. Make it the exact right size.  The simplest way is probably to create a list, initially empty, and then add only real advisors to it.